mapit_labour/urls.py

```diff
@@ -13,20 +13,20 @@
 format_end = r"(?:\.(?P<format>html|json))?"
 
 urlpatterns = [
-    url(r"^uprn/(?P<uprn>[0-9]+)%s$" % format_end, uprn, name="mapit_labour-uprn"),
-    url(r"^addressbase$", addressbase, name="mapit_labour-addressbase"),
-    url(r"^import/csv/?$", import_csv, name="mapit_labour-import_csv"),
-    url(
+    re_path(r"^uprn/(?P<uprn>[0-9]+)%s$" % format_end, uprn, name="mapit_labour-uprn"),
+    re_path(r"^addressbase$", addressbase, name="mapit_labour-addressbase"),
```

Comment:
You could also import path and make these use the simpler one, e.g. this one is `path('addressbase')`